### PR TITLE
test for cli load and save, #1387

### PR DESCRIPTION
**Pull request by bylsmad**
xfail tests for #1402
fix for help/welcome/Home.meta size
altering --path parameter on load-help
    now works same as dump-help
changing -a to flag for save
    appears this is how it is expected to work in quickinstall.py
fix for save with no -f option use stdout.buffer to output binary data

### CODE REVIEW

File: src/moin/cli/_tests/test_serialization.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            name = item[NAME][0]
+            content = item.pop(CONTENT)
+            metas[name] = item
+        assert set(expected_metas.keys()) == set(metas.keys())
+        for name, meta in metas.items():
+            expected_meta = expected_metas[name]
+            for k, v in meta.items():
+                if k in expected_meta:
+                    if k == MTIME:
+                        continue  # load replaces all mtimes
```
Comment (bylsmad) [inline src/moin/cli/_tests/test_serialization.py:81]: this seems a bit strange to me, is load from backup supposed to reset all the mtimes?

File: src/moin/cli/maint/modify_item.py [hunk 2]
```diff
@@ -144,8 +145,10 @@ def LoadHelp(namespace, path_to_help):
     Load an entire help namespace from distribution source.
     """
     logging.info("Load help started")
-    abspath_to_here = os.path.dirname(os.path.abspath(__file__))
-    path_to_items = os.path.normpath(os.path.join(abspath_to_here, path_to_help, namespace))
+    if path_to_help is None:
+        abspath_to_here = os.path.dirname(os.path.abspath(__file__))
+        path_to_help = os.path.abspath(os.path.join(abspath_to_here, '../../help/'))
+    path_to_items = os.path.normpath(os.path.join(path_to_help, namespace))
```
Comment (bylsmad) [inline src/moin/cli/maint/modify_item.py:151]: making this work same as dump-help

File: src/moin/help/welcome/Home.meta [hunk 3]
```diff
@@ -22,7 +22,7 @@
   "rev_number": 1,
   "revid": "be7556a9a778481c938ff656fe230c9d",
   "sha1": "59f6c196b7ff2db96cc620154e7c9addf46c33a4",
-  "size": 307,
+  "size": 324,
```
Comment (bylsmad) [inline src/moin/help/welcome/Home.meta:25]: this is the correct size

File: src/moin/cli/maint/modify_item.py (hunk 2 above)
Comment (UlrichB22) [reply src/moin/cli/maint/modify_item.py:151]: Perhaps you can try the same logic using moin_help as in 
https://github.com/moinwiki/moin/blob/bbd6ce2e9526ac8852b2b7f89d97d7c7c82e3225/src/moin/cli/maint/modify_item.py#L216
For me, it's easier to understand. What do you think about it?

Comment (bylsmad) [reply src/moin/cli/maint/modify_item.py:155]: @UlrichB22 done, see https://github.com/moinwiki/moin/pull/1418/commits/51ce36dd344f43509028fb51837c92cdbe2d9e9b

**Pull request merged**